

T_base_sensor:
 - [1, 0, 0, 0.3]
 - [ 0, 1, 0, 0]
 - [ 0,  0, 1, 0]
 - [ 0.0, 0.0, 0.0, 1.0]
 
label_path: /home/lukas/catkin_ws/src/panoptic_mapping/panoptic_mapping_ros/data/flat_dataset/labels.csv

tsdf_voxel_size: 0.1

tsdf_integrator:
  integrator_type: projective_multi_tsdf # naive
  horizontal_resolution: 640
  vertical_resolution: 480
  vertical_fov_deg: 73.7397952
  min_ray_length_m: 0.1
  max_ray_length_m: 5
  voxel_carving_enabled: false
  use_weight_dropoff: true


#  truncation_distance: 0.2
#  max_ray_length_m: 5.0
#  use_const_weight: false
#  use_sparsity_compensation_factor: true
#  sparsity_compensation_factor: 20.0
#  integration_order_mode: "sorted"
  
visualization:
  coloring_mode: color # submaps instances, color, normals

